src/DashboardHub/Bundle/AppBundle/Service/TravisService.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace DashboardHub\Bundle\AppBundle\Service;
+
+use GuzzleHttp\Client;
+use Tedivm\StashBundle\Service\CacheService as Cache;
+
+/**
+ * Class TravisService
+ * @package Quickstart\Bundle\AppBundle\Service
+ */
+class TravisService
+{
+    /**
+     * @var Client
+     */
+    private $client;
+    /**
+     * @var Cache
+     */
```

Comment:
done - changed visibility.

I thought of that, but this is probably going to get a refactor again soon.